### PR TITLE
Add release notes for 2.7.0

### PR DESCRIPTION
**Pull request by RyanL1997**
### Description
Add release notes for 2.7.0
* Category (Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation)
Maintenance

### Issues Resolved
* Relate https://github.com/opensearch-project/security/issues/2635

Is this a backport? If so, please add backport PR # and/or commits #

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by RyanL1997** [comment]
This pull request is not ready to be merged until the merge of https://github.com/opensearch-project/security/pull/2689

File: release-notes/opensearch-security.release-notes-2.7.0.0.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* Fix error message when system index is blocked ([#2525](https://github.com/opensearch-project/security/pull/2525))
+* Fix of OpenSSLTest is not using the OpenSSL Provider ([#2301](https://github.com/opensearch-project/security/pull/2301))
+* Add chmod 0600 to install_demo_configuration bash script ([#2550](https://github.com/opensearch-project/security/pull/2550))
+* Fix SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder" ([#2564](https://github.com/opensearch-project/security/pull/2564))
+* Fix lost privileges during auto initializing of the index ([#2498](https://github.com/opensearch-project/security/pull/2498))
+* Fix NPE and add additional graceful error handling ([#2687](https://github.com/opensearch-project/security/pull/2687))
+
+### Enhancements
+
+* Flatten response times ([#2471](https://github.com/opensearch-project/security/pull/2471))
```
Comment (cwperks) [inline release-notes/opensearch-security.release-notes-2.7.0.0.md:17]: This change was included in 2.6.0, but it looks like it was not added in the release notes. Can you take it out of this file and add it into the release note for 2.6?

**Review by scrawfor99 — APPROVED** [review]
Thank you Ryan!

File: release-notes/opensearch-security.release-notes-2.7.0.0.md (hunk 1 above)
Comment (RyanL1997) [reply release-notes/opensearch-security.release-notes-2.7.0.0.md]: Good catch! Done

**Review by cwperks — DISMISSED** [review]

**Review by scrawfor99 — DISMISSED** [review]

**Review by DarshitChanpura — APPROVED** [review]

**Review by scrawfor99 — APPROVED** [review]
Thank you Ryan!

**Review by cwperks — APPROVED** [review]

**Pull request merged**